_rules/image-no-text-0va7u6.md

```diff
@@ -36,24 +36,23 @@ This rule applies to any [visible][] [embedded image][].
 
 For the rendered image resource of the [image sources][] of each test target, at least one of the following is true:
 
-- the image resource contains text and it is [essential][] that the text is rendered with that specific presentation; or
-- the image resource does not contain text expressing anything in a [human language][]; or
-- the image resource contains text and the text is not a significant part of the visual content in the image; or
+- the [visible pixels][visible] of the rendered image resource contain text and it is [essential][] that the text is rendered with that specific presentation; or
```

Comment:
This seems like it could be simplified:

```suggestion
- the [visible pixels][visible] of the rendered image resource contain text for which its presentation is [essential]; or
```